meta-cli/src/cli/list.rs

```diff
@@ -0,0 +1,159 @@
+// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+// SPDX-License-Identifier: MPL-2.0
+
+use super::serialize::orchestrate_serialization_workflow;
+use super::{Action, ConfigArgs};
+use crate::config::NodeConfig;
+use crate::deploy::actors::task_manager::TaskSource;
+use crate::interlude::*;
+use anyhow::Result;
+use async_trait::async_trait;
+use clap::Parser;
+use common::graphql::Query;
+use common::typegraph::Typegraph;
+use reqwest::Url;
+use serde::Deserialize;
+
+#[derive(Parser, Debug)]
+pub struct List {
+    #[clap(long)]
+    max_parallel_loads: Option<usize>,
+}
+
+#[async_trait]
+impl Action for List {
+    #[tracing::instrument]
+    async fn run(&self, args: ConfigArgs) -> Result<()> {
+        let dir = args.dir()?;
+        let task_source = TaskSource::Discovery(dir.clone().into());
+        let typegraphs = orchestrate_serialization_workflow(
```

Comment:
Why not a new `TaskAction` instead of the `SerializeAction`? This would skip all the wasm calls...